package.json

```diff
@@ -2,27 +2,23 @@
   "name": "metadecidim",
   "private": true,
   "dependencies": {
-    "@decidim/browserslist-config": "^0.27.3",
-    "@decidim/core": "^0.27.3",
-    "@decidim/elections": "^0.27.3",
-    "@decidim/webpacker": "^0.27.3"
+    "@decidim/browserslist-config": "~0.28.0-rc2",
```

Comment:
maybe we need to update the versions to "~0.28.0-rc3" ?

Reply:
Done in 5b7c67d3642443bc4a1e9a1ab385b534d36e5d2e